### PR TITLE
Sensitive information to .env with python-decouple

### LINKED ISSUE #7: Separate sensitive information with help of python-decouple
**Is your feature request related to a problem? Please describe.**
Sensitive information like secret key, db credentials, extra should not be placed in the github repo. 

**Describe the solution you'd like**
1. With the help of python-decouple pull out the sensitive information into .env file.
1. Gitignore the .env file
1. update requirements.txt file

**Things to be put in the .env file:**
1. SECRET_KEY - string
1. DEBUG - cast to bool
1. ALLOWED_HOSTS - cast to list of strings

**Additional context**
We can make use of casting funcitonality included with the package.
https://github.com/henriquebastos/python-decouple#id13

### PR DESCRIPTION
**Pull request by akhil-maker**
### Description
I have added a file .env in .gitignore, updated settings.py and .gitignore.

**Type of Change:** (Delete irrelevant options)
- Code
- Testing
- User Interface

**Code/Quality Assurance Only** (Delete irrelevant options)

- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

### Checklist:
- [ ] I have performed a self-review of my own code or materials
- [ ] My PR follows the style guidelines of this project
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged
- [ ] Update Swagger documentation and the exported file at /docs folder
- [ ] Update requirements.txt

**Code/Quality Assurance**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
+python-decouple==3.3
 pytz==2019.3
+scikit-learn==0.22
+scipy==1.3.3
+shutilwhich==1.1.0
+sklearn==0.0
+SQLAlchemy==1.3.13
 sqlparse==0.3.1
+WebOb==1.8.6
+Werkzeug==1.0.0
```
Comment (naveennvrgup) [inline requirements.txt]: @akhil-maker please dont delete exisiting dependencies and dont add unnecessary dependencies. This commit needs only python-decouple to be added

File: server/settings.py [hunk 2]
```diff
@@ -11,6 +11,7 @@
 """
 
 import os
+from decouple import config
```
Comment (naveennvrgup) [inline server/settings.py:14]: nice on job on this file 🦊

**Review by naveennvrgup — COMMENTED** [review]
1. please address the comments
1. add .env to the .gitignore file
1. dont make PR from dev branch. for every issue.
    1. checkout dev and pull from upstream
    1. create a new branch to work on it 
1. work on virtual environment not in global environment
1. update README.md file on how to use the .env file

File: requirements.txt (hunk 1 above)
Comment (akhil-maker) [reply requirements.txt]: Yes sir

File: server/settings.py (hunk 2 above)
Comment (akhil-maker) [reply server/settings.py:14]: Thank you sir

File: requirements.txt [hunk 3]
```diff
@@ -1,7 +1,29 @@
 asgiref==3.2.7
+blinker==1.4
```
Comment (naveennvrgup) [inline requirements.txt]: @akhil-maker  dont add unnecessary dependencies. This commit needs only python-decouple to be added

**Comment by naveennvrgup** [comment]
Great job with your first contribution @akhil-maker. 🎈 🎊 . Here are a few things you can improve upon on your next PR:
1. Check [ X ] the box which applies to this PR.
image
1. instead of Fixes `#issue_number #33` do `Fixes #33`

**Pull request merged**